src/components/common/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+// Components
+import { TabItemType } from 'components/OrderBuySell'
+import TabItem from 'components/common/Tabs/TabItem'
+import TabContent from 'components/common/Tabs/TabContent'
+
+interface Props {
+  tabItems: TabItemType[]
+}
+
+const Tabs: React.FC<Props> = (props) => {
+  const [activeTab, setActiveTab] = useState<number>(1)
+
+  const onTabClick = (id: number): void => {
+    return setActiveTab(id)
+  }
+
+  const Wrapper = styled.div`
```

Comment:
never write styled components inside other components! they are remade on every render.

Reply:
Oooh good call, will engrave that in my brain from now on.

Reply:
Good eye, I didn't even notice

Reply:
Thanks, eye read carefully :eye: